### ISSUE TITLE
MERLIN : Jaws constantly being moved [TIMEBOX: 2d]

### ISSUE THREAD

**Issue by GDH-ISIS** [ISISComputingGroup/IBEX#3453]
During cycle, there was a problem with the MERLIN jaws. The Jaw amplifiers were eventually turned off as some of the jaws were creeping. Since this time, sample environment have taken the jaws out and cleaned them up. They looked fine on the bench. They asked for the jaws to be tested via software before they were re-installed.
On looking at the jaws, they look to be operational. However, there is a software problem, whereby the jaws are constantly being moved. The set points are being constantly updated and the motors are being moved - constantly twitching. .CNEN is changing between enabled and disabled a lot. Changing the retry dead-band RDBD appeared to have no effect.

**Comment by GDH-ISIS** [comment]
This look possibly related to : https://github.com/ISISComputingGroup/IBEX/issues/1178

**Comment by FreddieAkeroyd** [comment]
I've looked at MERLIN just now and though the readback is changing the SP is unchanged, has something else changed?

**Comment by GDH-ISIS** [comment]
In meeting sample environment at MERLIN to exercise the jaws and make sure they are ok, these were my observations. I did turn the jaws off as they were constantly being moved once every few seconds. I altered a few of the settings to try and solve the problem. However, I had no success and simply put the settings back. The jaws themselves did appear to be moving ok when asked to go to a new position. It was just the constant "twitching" at the set point and the set point changing that looked to be the problem.

**Comment by FreddieAkeroyd** [comment]
@GDH-ISIS turned the motors back on and I increased the MTR0101_EDEL_SP value from 0.055 to 0.2 which looks to have cured the problem. The MTR0101.ADEL value was also 0.055 which I have increased to 0.2 to stop excessive archiving. The Galil IOC does not have a PV that directly reflects motor on/off but from looking at _MOA on a galil command line it would appear the motors are off, monitoring of CNEN also does not change. The readback is fluctuating as expected, this could be stopped by increasing MDEL but isn't a problem and i'm not sure that is a requirement?
